experiments/calibration/generate_observations.jl

```diff
@@ -161,6 +161,7 @@ function preprocess_single_era5_var(var::OutputVar, short_name, nelements)
 end
```

Comment:
A note that I got an error running this as part of the job on GPU - it said it I needed to call the clima comms function import required backends.